deploy/charts/external-secrets/values.yaml

```diff
@@ -7,10 +7,12 @@ image:
   repository: ghcr.io/external-secrets/external-secrets
   pullPolicy: IfNotPresent
   # -- The image tag to use. The default is the chart appVersion.
+  tag: ""
+  # -- The flavour of tag you want to use
   # There are different image flavours available, like distroless and ubi.
   # Please see GitHub release notes for image tags for these flavors.
   # By default the distroless image is used.
-  tag: ""
+  flavour: ""
```

Comment:
In here you can add comments to the top, this way `make helm.docs` will generate the appropriate text to `charts/README.md`

Reply:
Updated docs, I was using podman and it caused some permission issues, but it is all good now

Reply:
@gusfcarvalho sorry to at you but did you have any other comments?